app/src/examples/AnimatedSensorGyroscopeExample.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Pressable, SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import Animated, {
+  SensorType,
+  useAnimatedSensor,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+} from 'react-native-reanimated';
+
+function clampWorklet(num: number, min: number, max: number) {
+  'worklet';
+  return Math.min(Math.max(num, min), max);
+}
+
+const OFFSET_X = 100; //in px
+const OFFSET_Y = 100; //in px
+const OFFSET_Z = 180; //in degrees
+
+export default function AnimatedSensorGyroscopeExample() {
+  const animatedSensor = useAnimatedSensor(SensorType.GYROSCOPE);
+
+  const xOffset = useSharedValue(-OFFSET_X);
+  const yOffset = useSharedValue(0);
+  const zOffset = useSharedValue(0);
+
+  const animatedStyles = useAnimatedStyle(() => {
+    const { x, y, z } = animatedSensor.sensor.value;
+    // The x vs y here seems wrong but is the way to make it feel right to the user
```

Comment:
What exactly seems to be wrong?

Reply:
We are using `y` gyroscope value modify `xOffset` and `x` value to modify `yOffset`, that may be unintuitive for the first sight.